<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HierarchyWindow</class>
 <widget class="QDialog" name="HierarchyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hierarchy</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>141</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="HierarchyLayout">
    <property name="spacing">
     <number>2</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>441</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="InspectorLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="NodeNameLabel">
      <property name="text">
       <string>NodeName</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="TransformComponent">
      <property name="text">
       <string>TransformComponent</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="positionXLabel">
      <property name="text">
       <string>PositionX</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="positionXLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="positionYLabel">
      <property name="text">
       <string>PositionY</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="positionYLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="sizeXLabel">
      <property name="text">
       <string>SizeX</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="sizeXLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="sizeYLabel">
      <property name="text">
       <string>SizeY</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="sizeYLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="scaleXLabel">
      <property name="text">
       <string>ScaleX</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="scaleXLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="scaleYLabel">
      <property name="text">
       <string>ScaleY</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="scaleYLineEdit"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="SpriteComponent">
      <property name="text">
       <string>SpriteComponent</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="imagePathLabel">
      <property name="text">
       <string>ImagePath</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="imagePathLineEdit"/>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="ClassParameters">
      <property name="text">
       <string>ClassParameters</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="specialParameterLabel">
      <property name="text">
       <string>SpecialParameter</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="specialParameterLineEdit"/>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveAllButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save all</string>
   </property>
  </widget>
  <widget class="QLabel" name="spriteComponentImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
